src/booking/Offers/OfferTypes.ts

```diff
@@ -291,6 +296,23 @@ export interface PaymentRequirements {
   requires_instant_payment: boolean
 }
 
+export interface OfferPrivateFare {
+  /**
+   * The corporate code that was applied, if any.
+   */
+  corporate_code?: string
```

Comment:
Note: These two fields (`corporate_code` and `tracking_reference`)  are 'required' in the docs, this appears to be a mistake as they are not always returned from the API.